src/components/metadata/field.js

```diff
@@ -87,32 +100,41 @@ class MetadataField extends React.PureComponent {
         className={cx(classes)}
         onContextMenu={this.handleContextMenu}>
         <label title={details.join('\n\n')}>{label}</label>
-        <div className="value" onClick={this.handleClick}>
-          <Editable
-            value={this.props.text}
-            getCompletions={getMetadataCompletions}
-            display={auto(this.props.text, this.props.type)}
-            placeholder={this.props.placeholder}
-            isActive={this.props.isEditing}
-            isRequired={this.props.isRequired}
-            onCancel={this.handleCancel}
-            onChange={this.handleChange}
-            onKeyDown={this.handleKeyDown}/>
-          {isInvalid && <IconWarningSm/>}
-          {this.props.isReadOnly && <IconLock/>}
-        </div>
+        {this.connect(
+          <div
+            className={cx('value', { over: this.props.isOver })}
+            onClick={this.handleClick}>
+            <Editable
+              value={this.props.text}
+              getCompletions={getMetadataCompletions}
+              display={auto(this.props.text, this.props.type)}
+              placeholder={this.props.placeholder}
+              isActive={this.props.isEditing}
+              isRequired={this.props.isRequired}
+              onCancel={this.handleCancel}
+              onChange={this.handleChange}
+              onKeyDown={this.handleKeyDown}/>
+            {isInvalid && <IconWarningSm/>}
+            {this.props.isReadOnly && <IconLock/>}
+          </div>)}
       </li>
     )
   }
 
-
   static propTypes = {
+    id: oneOfType([
+      number,
+      arrayOf(number)
+    ]),
+
     isEditing: bool,
     isDisabled: bool,
+    isDragging: bool,
```

Comment:
Do we need this? I think it's not used at the moment so I would remove it (or do you want to style the field being dragged @flachware)